code/modules/mob/living/simple_animal/hostile/dark_wizard.dm

```diff
@@ -30,6 +30,7 @@
 	maxbodytemp = INFINITY
 	atmos_requirements = list("min_oxy" = 0, "max_oxy" = 0, "min_tox" = 0, "max_tox" = 0, "min_co2" = 0, "max_co2" = 0, "min_n2" = 0, "max_n2" = 0)	
 	loot = list(/obj/effect/decal/remains/human)
+	del_on_death = 1
```

Comment:
TRUE

Reply:
everything else does 1 and not TRUE :flushed:

Reply:
you set it to TRUE because its a boolean value and it makes it more readable.

if something else isnt then it’s wrong or not a boolean.

Reply:
TRUE and FALSE are recent that is why. Its a better standard too